SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/ConnectionConfig.h

```diff
@@ -129,6 +129,7 @@ class FDevAuthConfig : public FConnectionConfig
 	{
 		UseExternalIp = true;
 		LocatorHost = SpatialConstants::LOCATOR_HOST;
```

Comment:
Oh, I forgot, shouldn't this branch between LOCATOR_HOST and LOCATOR_HOST_CN as FLocatorConfig does?

Reply:
oh yeah. will add that :)